server/prisma/schema.prisma

```diff
@@ -8,28 +8,121 @@ datasource db {
   url      = "file:./dev.db"
 }
 
-// This comes with the boilerplate.
 model User {
-  id                  String          @id @default(cuid())
-  createdAt                        DateTime        @default(now())
-  updatedAt                        DateTime        @updatedAt
-  email           String  @unique
-  password      String
-  role          String
-  lastLogin     DateTime?  
-  // LL: https://www.smashingmagazine.com/2017/11/safe-password-resets-with-json-web-tokens/
-  resetPasswordToken String
-  dateResetPasswordRequest DateTime?
-  validateEmailToken String
-  isEmailValidated Boolean
-  name  String
+  id                         Int                         @id @default(autoincrement())
+  fullName                   String?
+  email                      String                      @unique
+  password                   String
+  feeds                      Feed[]
+  boards                     Board[]
+  userSourceFavorites        UserSourceFavorite[]
+  userActivatedSubscriptions UserActivatedSubscription[]
+  createdAt                  DateTime                    @default(now())
+  setting                    Setting?
```

Comment:
settings (usually this word is plural in english, unless you refer to a specific "setting")